package.json

```diff
@@ -4,7 +4,7 @@
   "description": "OpenPaaS suite Webmail (https://open-paas.org/)",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start test/config/karma.conf.js",
+    "test": "LANG=C karma start test/config/karma.conf.js",
```

Comment:
Can you explain why? And this deserves a dedicated commit, IMO ;)

Reply:
ah, Crap, your eyes are everywhere !
My Linux system is using FR language, so the launched firefox also, and it makes some tests break, because we check the result (HTML), and compare it using EN language.

Reply:
commits split